### PR TITLE
Mes 8701 candidate previous next bug fix

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Fixed a bug where the new display would not appear on unsubmitted test and test centre journal.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

### CODE REVIEW

File: src/components/test-slot/test-slot/test-slot.html [hunk 1]
```diff
@@ -138,7 +138,7 @@
 
 <ng-template #candidateLink>
     <candidate-link *ngIf="canViewCandidateDetails() && !isTestCentreJournalADIBooking()"
-                    [isPracticeMode]="isPracticeMode"
+                    [slots]="slots"
```
Comment (matthew2564) [inline src/components/test-slot/test-slot/test-slot.html:141]: Any reason for removing the `isPracticeMode` input?

File: src/app/pages/unuploaded-tests/unuploaded-tests.page.ts [hunk 2]
```diff
@@ -57,12 +59,14 @@ export class UnuploadedTestsPage implements OnInit {
             new Date(a.slotData.slotDetail.start).getTime() - new Date(b.slotData.slotDetail.start).getTime())),
       ),
     };
+    this.pageState.unSubmittedTestSlots$.subscribe((data) => {
+      this.slots = data.map((i) => i.slotData);
+    });
```
Comment (matthew2564) [inline src/app/pages/unuploaded-tests/unuploaded-tests.page.ts]: We should add this into the `this.pageState` and using the async pipe inside the template as it auto manages subscriptions.

File: src/app/pages/unuploaded-tests/unuploaded-tests.page.ts [hunk 3]
```diff
@@ -29,6 +30,7 @@ interface UnunploadedTestsPageState {
 })
 export class UnuploadedTestsPage implements OnInit {
   pageState: UnunploadedTestsPageState;
+  slots: TestSlot[];
```
Comment (matthew2564) [inline src/app/pages/unuploaded-tests/unuploaded-tests.page.ts]: This should be moved into the `UnunploadedTestsPageState`

File: src/app/pages/test-centre-journal/components/view-journals-card/view-journals-card.ts [hunk 4]
```diff
@@ -131,15 +132,19 @@ export class ViewJournalsCardComponent implements OnChanges {
   };
 
   onShowJournalClick = (): void => {
+
     this.hasClickedShowJournal = true;
+
     if (!this.journal) {
+      console.log('not journal');
```
Comment (matthew2564) [inline src/app/pages/test-centre-journal/components/view-journals-card/view-journals-card.ts]: Please remove

File: src/components/test-slot/test-slot/test-slot.html (hunk 1 above)
Comment (RLCorp) [reply src/components/test-slot/test-slot/test-slot.html:141]: was added in another PR for this feature, change in functionality meant it was no longer needed.

File: src/app/pages/unuploaded-tests/unuploaded-tests.page.ts (hunk 2 above)
Comment (RLCorp) [reply src/app/pages/unuploaded-tests/unuploaded-tests.page.ts]: updated in newest commit

File: src/app/pages/test-centre-journal/components/view-journals-card/view-journals-card.ts (hunk 4 above)
Comment (RLCorp) [reply src/app/pages/test-centre-journal/components/view-journals-card/view-journals-card.ts]: updated in newest commit

File: src/app/pages/unuploaded-tests/unuploaded-tests.page.ts (hunk 3 above)
Comment (RLCorp) [reply src/app/pages/unuploaded-tests/unuploaded-tests.page.ts]: updated in newest commit

**Review by matthew2564 — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**